js/scripts/util.js

```diff
@@ -125,7 +125,8 @@ const SFX = {
     COIN_COLLECT: {path: "./sfx/rune_collect.mp3", volume: 0.7},
     MEGA_MUSHROOM: {path: "./sfx/mega_mushroom.mp3", volume: 0.4},
     WATER_BALLOON: {path: "./sfx/water_balloon.wav", volume: 0.4},
-    LIFE_UP: {path: "./sfx/life_up.wav", volume: 0.4},
+    LIFE_UP: { path: "./sfx/life_up.wav", volume: 0.4 },
+    SWORD_UPGRADE: { path: "./sfx/sword_upgrade.mp3", volume: 0.4 },
```

Comment:
Would you be interested in updating the formatting so that it's all consistent in util.js? I'm referring to having a space before and after the data in the object next to the curly brackets. I don't care if we do or don't have a space, I am just not a fan of the inconsistency.